src/main/java/com/synopsys/integration/alert/channel/hipchat/descriptor/HipChatGlobalUIConfig.java

```diff
@@ -27,28 +27,24 @@
 
 import org.springframework.stereotype.Component;
 
-import com.synopsys.integration.alert.channel.hipchat.HipChatChannel;
 import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.PasswordConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
-import com.synopsys.integration.alert.common.descriptor.config.ui.DescriptorMetadata;
 import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
-import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
-import com.synopsys.integration.alert.common.enumeration.DescriptorType;
 import com.synopsys.integration.alert.common.enumeration.FieldGroup;
 
 @Component
 public class HipChatGlobalUIConfig extends UIConfig {
 
-    public List<ConfigField> setupFields() {
-        final ConfigField apiKey = PasswordConfigField.createRequired(HipChatDescriptor.KEY_API_KEY, "Api Key");
-        final ConfigField hostServer = TextInputConfigField.createGrouped(HipChatDescriptor.KEY_HOST_SERVER, "Host Server", FieldGroup.ADVANCED);
-        return List.of(apiKey, hostServer);
+    public HipChatGlobalUIConfig() {
+        super("HipChat", "hipchat", "comments");
```

Comment:
Encapsulate lowercase hipchat and comments strings into static final strings in the descriptor class since they are the URL and font-awesome icon name.